### PR TITLE
2693 - do not use contact number in the post

### PR DESCRIPTION
**Pull request by rowasc**
This pull request makes the following changes:
- Contact's posts are saved with form name + random number as their title 

Test checklist:
- [x] Create a set of contacts for a form
- [x] Verify that the posts created for each contact match the format "{Form title} - {random number}"

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform#2693

Ping @ushahidi/platform

### CODE REVIEW

File: application/classes/Ushahidi/Listener/ContactListener.php [hunk 1]
```diff
@@ -60,6 +60,19 @@ public function setTargetedSurveyStateRepo(TargetedSurveyStateRepository $repo)
 		$this->targeted_survey_state = $repo;
 	}
 
+	// CreateRepository
+	private function getContactPostTitle($form_name, $contact_id)
+	{
+		try {
+			$uuid = substr(Ramsey\Uuid\Uuid::uuid4()->toString(), 0 , 5);
```
Comment (rowasc) [inline application/classes/Ushahidi/Listener/ContactListener.php]: @rjmackay we need to generate a random number that's around 4-6 characters long to use in the post title for posts that belong to a contact (just so they don't all have the same title). Ideas on a nicer way to do that?

Comment (rjmackay) [reply application/classes/Ushahidi/Listener/ContactListener.php]: http://php.net/manual/en/function.random-int.php <- requires PHP 7 though.
But there's a userland implementation too https://github.com/paragonie/random_compat (we might already be using that I think)

Comment (rjmackay) [reply application/classes/Ushahidi/Listener/ContactListener.php]: PHP has finally passed the bar of simple and safe crypto methods... its like a real programming language 😆

Comment (rowasc) [reply application/classes/Ushahidi/Listener/ContactListener.php]: ha! thanks I didn't realize that was a thing

**Comment by rowasc** [comment]
@willdoran can you take a look? I'm using the library Robbie suggested now

**Pull request merged**